src/util/svg-text-wrapper.js

```diff
@@ -0,0 +1,99 @@
+const TextWrapper = require('./text-wrapper');
+
+/**
+ * Measure text by using a hidden SVG attached to the DOM.
+ * For use with TextWrapper.
+ */
+class SVGMeasurementProvider {
+    constructor () {
+        this._svgRoot = null;
+    }
+
+    /**
+     * Detach the hidden SVG element from the DOM and forget all references to it and its children.
+     */
+    dispose () {
```

Comment:
@cwillisf I noticed this method is never called by the text wrapper and also isn't called in the `endMeasurementSession` method. Seems like maybe a relic of a previous API? Where should it be called?

(Found this out when I was poking around the DOM and noticed that there were a bunch of suspicious SVG's attached to the end of the DOM!)

Reply:
What I had in mind is that this would be called only when the `SVGMeasurementProvider` is never going to be used again -- not after every measurement session. I probably should have hooked it up to a `dispose()` method on `SVGTextWrapper`, to be called by whoever owns the text wrapper.

Calling `dispose()` at the end of each measurement session is OK for now I think, but I imagined adding & removing an SVG node on the DOM to be a relatively expensive operation so I wanted to minimize the number of times it happens. [Premature optimization](http://wiki.c2.com/?PrematureOptimization), maybe :)